src/Compilers/Core/CodeAnalysisTest/ShadowCopyAnalyzerAssemblyLoaderTests.cs

```diff
@@ -10,7 +10,7 @@
 using Xunit;
 
 // The ShadowCopyAnalyzerAssemblyLoader type is only defined for the below platforms
-#if NET472 || NETCOREAPP2_1
+#if NET472 || NETCOREAPP3_1
```

Comment:
💭 Do we run this anywhere else?

Reply:
Not sure what you mean by this. Can you elaborate a bit?

Reply:
I'm asking if the `#if` necessary here. It seems to be enabled for all supported TFMs.

Reply:
Ah I see. Yeah this doesn't seem to make much sense now.